openhands/server/session/manager.py

```diff
@@ -97,27 +108,37 @@ async def _redis_subscribe(self):
     async def _process_message(self, message: dict):
         data = json.loads(message['data'])
         logger.debug(f'got_published_message:{message}')
-        sid = data['sid']
         message_type = data['message_type']
         if message_type == 'event':
+            sid = data['sid']
             session = self._local_agent_loops_by_sid.get(sid)
             if session:
                 await session.dispatch(data['data'])
         elif message_type == 'is_session_running':
             # Another node in the cluster is asking if the current node is running the session given.
-            session = self._local_agent_loops_by_sid.get(sid)
-            if session:
+            rid = data['rid']
```

Comment:
what is a `rid`? can we give it a better name?

Reply:
I've converted `rid` -> `request_id`